Source/Core/DolphinWX/FrameTools.cpp

```diff
@@ -879,7 +880,7 @@ void CFrame::DoStop()
     if (NetPlayDialog::GetNetPlayClient())
       NetPlayDialog::GetNetPlayClient()->Stop();
 
-    if (!m_tried_graceful_shutdown && TriggerSTMPowerEvent())
+    if (!m_tried_graceful_shutdown && UICommon::TriggerSTMPowerEvent(m_use_debugger))
     {
       m_tried_graceful_shutdown = true;
```

Comment:
The old `TriggerSTMPowerEvent` sets `m_confirm_stop = false`, but not the new one.